mep/urls.py

```diff
@@ -5,7 +5,7 @@
 from django.conf.urls import include, url
 from django.conf.urls.static import serve
 from django.contrib import admin
-from django.views.generic.base import RedirectView
+from django.views.generic.base import RedirectView, TemplateView
 from wagtail.admin import urls as wagtailadmin_urls
 from wagtail.contrib.sitemaps import Sitemap
```

Comment:
super minor, but it might look a little cleaner to import this `as WagtailSitemap` or something to avoid confusion